packages/ui/src/composites/wui-transaction-list-item/index.ts

```diff
@@ -0,0 +1,83 @@
+import { html, LitElement } from 'lit'
```

Comment:
Imports looking a bit chaotic imo. 

@0xAsimetriq What about using Trivago's import sorter? I'm using it for almost every project and love it.

- https://github.com/trivago/prettier-plugin-sort-imports

Reply:
+1 imo

Reply:
So, we actually tried this through prettier / as eslint rules and it created more issues than it solved, people didn't have certain plugins installed / reorganizing these caused a bunch of merge conflicts etc. So for now lets keep import and prop ordering relaxed. The only thing that we try to stick with is separate type imports i.e. using `import type {} ...` this will be caught by tsc.